chef/data_bags/crowbar/bc-template-glance.schema

```diff
@@ -76,6 +76,7 @@
             "rbd": {
               "type": "map", "required": true, "mapping": {
                 "store_ceph_conf": { "type": "str", "required": true },
+                "store_admin_keyring": { "type": "str", "required": true },
```

Comment:
@vuntz Do you have any opinion on this? As said I am ok with it, but I don't especially like it.